eduvpn/interface/error.py

```diff
@@ -0,0 +1,7 @@
+
+def translate_error(error: Exception) -> str:
```

Comment:
ik weet niet of 'translate' een goeie term is, aangezien we op een bepaald moment ook willen internationaliseren. Of is dat wat je hiermee van plan bent?

Reply:
ik merge m er wel alvast in, dan ben je niet geblockt.